runtime/src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // The version of the runtime specification. A full node will not attempt to use its native
     //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
-    spec_version: 1180,
+    spec_version: 1200,
```

Comment:
No needed after you remove useless changes. Anyway, we increment `spec_version` by ten, not more.

Reply:
Removed.